src/main/resources/locale/locale_de.properties

```diff
@@ -137,7 +137,7 @@ Commands.Description.mcability = Schalte die Bereitschaft von F\u00E4higkeiten b
 Commands.Description.mcchatspy = Schalte den Party Chat-Spy an oder aus.
 Commands.Description.mcconvert = Konvertiert Datenbanktypen oder Erfahrungsformeln.
 Commands.Description.mccooldown = Sieh alle F\u00E4higkeiten-Cooldowns.
-Commands.Description.mcgod = Schalte mcMMO godmode an oder aus.
+Commands.Description.mcgod = Schalte mcMMO Godmode an oder aus.
```

Comment:
Maybe this?
```suggestion
Commands.Description.mcgod = Schalte mcMMO Gottmodus an oder aus.
```

Reply:
As by my knowledge from playing on german servers, everyone says godmode. Never heard anyone refer to it as Gottmodus, even though that would be the correct translation. I'll take the liberty to leave it as Godmode, if that's okay ^^